src/components/Popover/hooks.ts

```diff
@@ -33,13 +33,13 @@ export const useDisableScroll = (
  * @param {Boolean} closeCallback - function that closes the popover
  */
 export const useCloseOnScroll = (
-  closeOnScrollRef: MutableRefObject<HTMLElement> | undefined,
+  closeOnScrollRef: RefObject<HTMLElement> | undefined,
```

Comment:
changed from `MutableRefObject` to `RefObject` because `current` is **not** being reassigned.